backend/src/bin/scripts/change-tenant-plan.ts

```diff
@@ -70,7 +71,7 @@ if (parameters.help || !parameters.tenant || !parameters.plan) {
     const tenantIds = parameters.tenant.split(',')
 
     for (const tenantId of tenantIds) {
-      const tenant = await options.database.tenant.findByPk(tenantId)
```

Comment:
This is not required to change but at the end of the file there is a `tenant.update` method call that needs to be handled in the TenantRepository.